DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -69,8 +69,9 @@ class PrivacyProtectionOverviewController: UITableViewController {
     }
     
     override func shouldPerformSegue(withIdentifier identifier: String, sender: Any?) -> Bool {
-        
-        if identifier == "Leaderboard" && !NetworkLeaderboard.shared.shouldShow() {
+
+        let level = TrackerNetworkLeaderboardView.Constants.minimumDetectionLevel
+        if identifier == "Leaderboard" && !NetworkLeaderboard.shared.shouldShow(minimumDetectionLevel: level) {
```

Comment:
This doesn't work for me: identifier is "" when navigating to Leaderboard.

Reply:
That isn't new logic though.